README.md

```diff
@@ -0,0 +1,583 @@
+[![jazzband][jazzband-image]][jazzband]
```

Comment:
I'm almost sure this should be something like
```suggestion
[![jazzband-image]][jazzband]
```

Reply:
But what's about the syntax for the [alt text](https://www.w3schools.io/file/markdown-images/#how-to-add-an-image-with-a-link)?

Reply:
The alt text is what's in the first pair of brackets, the second one is the link to the thing. If you want different alt text, you could change the link definition, I think. Also, Markdown has various extensions that may or may not be supported by different MD parsers. I'm pretty sure MyST is mostly bare-bones, except for its own addition for Sphinx/RST compat...